docs/cicd/github-ci-template.yml

```diff
@@ -0,0 +1,31 @@
+# This is just an example workflow for continous integration.
+# You should customize it to meet your own requirements.
+name: 'Continuous Integration'
+on:
+  # When pull requests targeting the dev branch created.
+  pull_request:
+    branches:
+      - dev
+jobs:
+  buildAndTest:
+    runs-on: ubuntu-latest
+    steps:
+      # Setup environment.
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      # Checkout the code.
+      - uses: actions/checkout@v2
+
+      # Build the project.
+      # The way to build the current project depends on how you scaffold it.
+      # Different folder structures require different commands set.
+      - name: Build the project
+        run: cd tabs && npm install && npm run build
```

Comment:
use `npm ci` instead of `install` in CI containers

Reply:
Better, update it later.

Reply:
later like in another PR? why not now?

Reply:
> later like in another PR? why not now?

No, in this PR, that means wip.